extensions/spring-xd-extension-sqoop/src/main/java/org/springframework/xd/sqoop/SqoopTasklet.java

```diff
@@ -57,6 +62,17 @@ protected boolean isStoppable() {
 		command.add("java");
 		command.add(SQOOP_RUNNER_CLASS);
 		command.addAll(Arrays.asList(arguments));
+		Iterator<PropertySource<?>> i = environment.getPropertySources().iterator();
+		while (i.hasNext()) {
+			PropertySource<?> p = i.next();
+			if (p instanceof EnumerablePropertySource) {
+				for (String name : ((EnumerablePropertySource)p).getPropertyNames()) {
+					if(name.startsWith(SPRING_HADOOP_CONFIG_PREFIX)) {
+						command.add(name + "=" + environment.getProperty(name));
```

Comment:
not sure what we would truncate? You mean remove that part? I think we want to keep it for now, so we can tell which ones are 'spring.hadoop.config' properties. We remove it in the SqoopRunner before we use the property .